source/installation-guide/wazuh-dashboard/step-by-step.rst

```diff
@@ -123,16 +123,131 @@ Starting the Wazuh dashboard service
 
     When you access the Wazuh dashboard for the first time, the browser shows a warning message stating that the certificate was not issued by a trusted authority. An exception can be added in the advanced options of the web browser or, for increased security, the ``root-ca.pem`` file previously generated can be imported to the certificate manager of the browser. Alternatively, a certificate from a trusted authority can be configured. 
 
+
+Securing your Wazuh installation
+--------------------------------
+
+Change the default credentials to protect your Wazuh solution. 
+
+Change the default Wazuh API credentials
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
![image](https://user-images.githubusercontent.com/3830293/169309005-86100cd4-1e9d-4c34-8e84-5c214481f5dd.png)

This section results in a little bit weird to me. Using a big title only for a sentence "Change the default credentials to protect your Wazuh solution."
I would opt for two options:
- Include a valuable paragraph, explaining that after a default installation with default credentials, bots and others can take default users and scrap the networking searching for vulnerable installations.
- If this paragraph doesn't add value to the users, I would remove the title and the mentioned sentence, leaving only subtitles with changing default credentials steps.

Reply:
I have expanded the paragraph to explain that once the installation of the central components is complete, we recommend changing the default credentials to avoid possible attacks.